test/validation_test.rb

```diff
@@ -76,26 +76,11 @@ class Measured::Rails::ValidationTest < ActiveSupport::TestCase
     assert_equal ["Length units singular custom message too"], thing.errors.full_messages
   end
 
-  test "validation presence works on measured columns" do
```

Comment:
No need to remove this, or any of the setup. It's good to prove that the existing rails presence validation continues to work.